sympy/functions/elementary/complexes.py

```diff
@@ -1000,6 +1181,15 @@ class principal_branch(Function):
     >>> principal_branch(exp_polar(2*pi*I)*3*z, 2*pi)
     3*principal_branch(z, 2*pi)
 
+    Parameters
+    ==========
+
+    x : Expr
+        Expression which can be an exponent or exponent representing a polar
+        number.
+
+    period : Positive real number.
```

Comment:
I don't think that these explanations are mathematically correct or informative. and it's just weird to see something like
`exponent or exponent representing a polar number`

I'd just suggest to rewrite these informal statements about parameters
```
    This is a function of two arguments. The first argument is a polar
    number `z`, and the second one a positive real number of infinity, `p`.
    The result is "z mod exp_polar(I*p)".
```
to the format that fits the parameters section, such like
```
x
    A polar number

period
    A positive real number of infinity
```
At least in the words that the original author had used

Reply:
Here we can't mention a certain datatype for the parameters then.

Reply:
It could be
```
period : Expr
    A positive real number of infinity
```

Reply:
Yes, I have done this now.